package-lock.json

```diff
@@ -7634,101 +7785,432 @@
+      "integrity": "sha512-t9TTLtPQxtQB3SAf/5E8xPXfVDsC6WGOsgKY02l2cbe0HLymT7ynE8Hu48Lk5qynHCquj6nhISfEHcjMkYpu/A==",
+      "dependencies": {
+        "@babel/runtime": "^7.23.4",
+        "@noble/curves": "^1.2.0",
+        "@noble/hashes": "^1.3.2",
+        "@solana/buffer-layout": "^4.0.1",
+        "agentkeepalive": "^4.5.0",
+        "bigint-buffer": "^1.1.5",
+        "bn.js": "^5.2.1",
+        "borsh": "^0.7.0",
+        "bs58": "^4.0.1",
+        "buffer": "6.0.3",
+        "fast-stable-stringify": "^1.0.0",
+        "jayson": "^4.1.0",
+        "node-fetch": "^2.7.0",
+        "rpc-websockets": "^7.5.1",
+        "superstruct": "^0.14.2"
+      }
     },
-    "node_modules/@trezor/connect-common": {
-      "version": "0.0.10",
-      "resolved": "https://registry.npmjs.org/@trezor/connect-common/-/connect-common-0.0.10.tgz",
-      "integrity": "sha512-bwqB5J6IrLBZ/tGQw/C/EoYZW6Jpjf6ch9sYUog4EVTNIqh5nCLP+g8ShSyco9lvrKInMDXDWwU6OsEZE5gcPQ==",
-      "dev": true
+    "node_modules/@trezor/blockchain-link-utils/node_modules/@types/node": {
+      "version": "12.20.55",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-12.20.55.tgz",
+      "integrity": "sha512-J8xLz7q2OFulZ2cyGTLE1TbbZcjpno7FaN6zdJNrgAdrJ+DZzh/uFR6YrTb4C+nXakvud8Q4+rbhoIWlYQbUFQ=="
     },
-    "node_modules/@trezor/rollout": {
-      "version": "1.3.2",
-      "resolved": "https://registry.npmjs.org/@trezor/rollout/-/rollout-1.3.2.tgz",
-      "integrity": "sha512-P660yMKCM0rKJZXEH+YmTYXLdrxcKquapSnC7YUa5cf0jVG8rNQLeETnaS6YnYjbYXRxtmUWL/537bfj6dZeWw==",
-      "dev": true,
+    "node_modules/@trezor/blockchain-link-utils/node_modules/jayson": {
+      "version": "4.1.0",
+      "resolved": "https://registry.npmjs.org/jayson/-/jayson-4.1.0.tgz",
+      "integrity": "sha512-R6JlbyLN53Mjku329XoRT2zJAE6ZgOQ8f91ucYdMCD4nkGCF9kZSrcGXpHIU4jeKj58zUZke2p+cdQchU7Ly7A==",
       "dependencies": {
-        "@trezor/utils": "^1.0.0",
-        "cross-fetch": "^3.1.5",
-        "runtypes": "^6.5.1"
+        "@types/connect": "^3.4.33",
+        "@types/node": "^12.12.54",
+        "@types/ws": "^7.4.4",
+        "commander": "^2.20.3",
+        "delay": "^5.0.0",
+        "es6-promisify": "^5.0.0",
+        "eyes": "^0.1.8",
+        "isomorphic-ws": "^4.0.1",
+        "json-stringify-safe": "^5.0.1",
+        "JSONStream": "^1.3.5",
+        "uuid": "^8.3.2",
+        "ws": "^7.4.5"
+      },
+      "bin": {
+        "jayson": "bin/jayson.js"
+      },
+      "engines": {
+        "node": ">=8"
       }
     },
-    "node_modules/@trezor/rollout/node_modules/@trezor/utils": {
-      "version": "1.0.1",
-      "resolved": "https://registry.npmjs.org/@trezor/utils/-/utils-1.0.1.tgz",
-      "integrity": "sha512-+XYu51Zc3eM6bc8e1C3xrGHpryj6HsI4Gy3fdjhs8qiYGlVBJ6yMOsuNB4k2JHQiBNyhSn6Jw+VztH+ZYps1kQ==",
-      "dev": true
+    "node_modules/@trezor/blockchain-link/node_modules/@solana/web3.js": {
+      "version": "1.89.1",
```

Comment:
Hmm, seems like we're going to be redundantly bringing in web3.js lib here. Is there any way to make the versions match?